content/hardware/02.hero/boards/uno-rev3/tutorials/PWM-using-MATLAB-Tutorial/PWM-using-MATLAB-Tutorial.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Controlling PWM output with a potentiometer on the Arduino® UNO R3 via MATLAB®'
```

Comment:
```suggestion
title: 'Controlling PWM Output with a Potentiometer via MATLAB®'
```

Fix content linter failures:

- Make title <=60 characters
  - My thinking is this tutorial is already categorized under the Arduino Uno board section, so that information is available to the reader in other ways
- Use [title case](https://en.wikipedia.org/wiki/Title_case)